doc/sphinxext/math_symbol_table.py

```diff
@@ -27,71 +34,42 @@
-     \perp \cong \prec \succ \mid \neq \preceq \succeq \parallel \sim
-     \ll \gg \bowtie \simeq \subset \supset \Join \approx \subseteq
-     \supseteq \ltimes \asymp \sqsubset \sqsupset \rtimes \doteq
-     \sqsubseteq \sqsupseteq \smile \propto \dashv \vdash \frown
-     \models \in \ni \notin \approxeq \leqq \geqq \lessgtr \leqslant
-     \geqslant \lesseqgtr \backsim \lessapprox \gtrapprox \lesseqqgtr
-     \backsimeq \lll \ggg \gtreqqless \triangleq \lessdot \gtrdot
-     \gtreqless \circeq \lesssim \gtrsim \gtrless \bumpeq \eqslantless
-     \eqslantgtr \backepsilon \Bumpeq \precsim \succsim \between
-     \doteqdot \precapprox \succapprox \pitchfork \Subset \Supset
-     \fallingdotseq \subseteqq \supseteqq \risingdotseq \sqsubset
-     \sqsupset \varpropto \preccurlyeq \succcurlyeq \Vdash \therefore
-     \curlyeqprec \curlyeqsucc \vDash \because \blacktriangleleft
-     \blacktriangleright \Vvdash \eqcirc \trianglelefteq
-     \trianglerighteq \neq \vartriangleleft \vartriangleright \ncong
-     \nleq \ngeq \nsubseteq \nmid \nsupseteq \nparallel \nless \ngtr
-     \nprec \nsucc \subsetneq \nsim \supsetneq \nVDash \precnapprox
-     \succnapprox \subsetneqq \nvDash \precnsim \succnsim \supsetneqq
-     \nvdash \lnapprox \gnapprox \ntriangleleft \ntrianglelefteq
-     \lneqq \gneqq \ntriangleright \lnsim \gnsim \ntrianglerighteq
-     \coloneq \eqsim \nequiv \napprox \nsupset \doublebarwedge \nVdash
-     \Doteq \nsubset \eqcolon \ne
-     """.split()],
+     _mathtext.Parser._relation_symbols],
     ["Arrow symbols",
      4,
-     r"""\leftarrow \longleftarrow \uparrow \Leftarrow \Longleftarrow
-     \Uparrow \rightarrow \longrightarrow \downarrow \Rightarrow
-     \Longrightarrow \Downarrow \leftrightarrow \updownarrow
-     \longleftrightarrow \updownarrow \Leftrightarrow
-     \Longleftrightarrow \Updownarrow \mapsto \longmapsto \nearrow
-     \hookleftarrow \hookrightarrow \searrow \leftharpoonup
-     \rightharpoonup \swarrow \leftharpoondown \rightharpoondown
-     \nwarrow \rightleftharpoons \leadsto \dashrightarrow
-     \dashleftarrow \leftleftarrows \leftrightarrows \Lleftarrow
-     \Rrightarrow \twoheadleftarrow \leftarrowtail \looparrowleft
-     \leftrightharpoons \curvearrowleft \circlearrowleft \Lsh
-     \upuparrows \upharpoonleft \downharpoonleft \multimap
-     \leftrightsquigarrow \rightrightarrows \rightleftarrows
-     \rightrightarrows \rightleftarrows \twoheadrightarrow
-     \rightarrowtail \looparrowright \rightleftharpoons
-     \curvearrowright \circlearrowright \Rsh \downdownarrows
-     \upharpoonright \downharpoonright \rightsquigarrow \nleftarrow
-     \nrightarrow \nLeftarrow \nRightarrow \nleftrightarrow
-     \nLeftrightarrow \to \Swarrow \Searrow \Nwarrow \Nearrow
-     \leftsquigarrow
-     """.split()],
+     _mathtext.Parser._arrow_symbols],
+    ["Dot symbols",
+     4,
+     r"""\cdots \vdots \ldots \ddots \adots \Colon \therefore \because""".split()],
```

Comment:
Aren't `therefore` and `because` in Relation operators?

Reply:
Yes, they are. As well.

I just saw some table with "dot symbols" somewhere (do not remember where, maybe the "List of LaTeX symbols..." or whatever it is called), where they were included, so thought that it may make sense. Easy to modify though.